File: Plov/src/Veg.java

public class Veg {
	

	private boolean isDirty = true;
	private boolean needCut=true;
	private int ready = 0;
	private boolean inPan;
	public boolean getInPan() {
		return inPan;
	}
	
	public void setInPan(boolean s) {
		inPan=s;
	}
	
	public void setDirty(boolean s) {
		isDirty = s;
	}
	
	public boolean getDirty(){
		return isDirty;
	}

	public void setNeedCut(boolean cut) {
		needCut = cut;
	}
	
	public boolean getNeedCut() {
		return needCut;
	}
	
	
	public int getReady() {
		return ready;
	}


	public void heat()
	{
		while (ready < 10) ready++;
	}
}

File: Plov/Plov/Veg.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Plov
{
    class Veg
    {

        private bool dirty=true;
        private bool cut = true;
        private bool ready = false;

        public bool isDirty
        {
            set
            {
                dirty = value;
            }

            get
            {
                return dirty;
            }
        }

        public bool needCut
        {
            set
            {
                cut = value;
            }

            get
            {
                return cut;
            }
        }


        public bool isReady
        {
            get
            {
                return ready;
            }
        }

        public void heat()
        {
            ready = true;
        }
    }
}
